src/main/java/Driver.java

```diff
@@ -2,33 +2,81 @@
 
 //import com.google.common.base.Strings;
 
+import main.java.Exceptions.NoParameterException;
+import main.java.Exceptions.OverwriteException;
+import main.java.Exceptions.WrongParameterException;
 import org.json.JSONArray;
+import org.json.JSONObject;
 
-public class Driver implements IDriver{
 
-    @Override
-    public void createDatabase(final String databaseSchemaPath) {
+public final class Driver implements IDriver {
+
+    private Executor executor;
 
+    /**
+     * @param executor
+     */
+    public Driver(final Executor executor) {
+        this.executor = executor;
     }
 
     @Override
-    public void setRow(final String databaseName, final String tableName, final String value) {
+    public void createDatabase(final String databaseSchemaPath) throws Exception {
+        String command = " -c create -sc " + databaseSchemaPath;
+        JSONObject api = executor.execute(command);
+        check(api);
+    }
 
+    @Override
+    public void setRow(final String databaseName, final String tableName, final String value) throws Exception {
+        String command = " -c set -db " + databaseName + " -t " + tableName + " -q " + value;
+        Executor executor = new Executor();
```

Comment:
You should use this.executor.

If u r creating new, so what is the benifits of passing executor ?